Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_Visualizer/Scripts/UserInput_Playback.cs

```diff
@@ -113,7 +113,11 @@ public void LoadNewFile(string filename)
 
             try
             {
+#if WINDOWS_UWP
```

Comment:
LoadNewFile () is only loaded in the Editor mode. So, the way it's written now, the following part should never be called: 
`#if WINDOWS_UWP`
`if (!UnityEngine.Windows.File.Exists(filename))`
`...`

See UserInput_Playback.Load() -> LoadInEditor() -> LoadNewFile(...). 

Is `!UnityEngine.Windows.File.Exists(filename)` causing the problem?

Reply:
Exactly - from this site:

https://docs.unity3d.com/ScriptReference/Windows.File.html

"This is an alternative for System.IO.File class. Note: this class is only available when targeting Universal Windows Platform.'

In reality we should probably have some file wrapper that abstracts away the platform-dependent options, but that's for another day.